docs/man_pages/project/testing/livesync-android.md

```diff
@@ -0,0 +1,35 @@
+livesync android
+==========
+
+Usage | Synopsis
+------|-------
+General | `$ tns livesync android [--device <Device ID>] [--companion] [--watch]`
+
+Synchronizes the latest changes in your project to devices. 
+
+`<Device ID>` is the device index or identifier as listed by run `$ tns device`
```

Comment:
This belongs to an `### Attributes` section. Usually, the `### Attributes` section is below the `### Options`.

Apply to other instances as needed.